core/templates/dev/head/pages/exploration_editor/ParamChangesEditorDirective.js

```diff
@@ -83,7 +83,8 @@ oppia.directive('paramChangesEditor', [function() {
               angular.copy(DEFAULT_PARAM_SPEC));
             $scope.paramNameChoices = generateParamNameChoices();
           }
-          $scope.paramChangesService.displayed.push(newParamChange);
+          $scope.paramChangesService.displayed.push(
+            angular.copy(newParamChange));
```

Comment:
Odd. I would actually expect the customizationArgs to be coupled because you're passing a reference to the dict in DEFAULT_CUSTOMIZATION_ARGS when you create a new default param change (i.e. there ought to be an angular.copy in ParamChange.createDefault. But generatorId is just a primitive string so I would have expected it to be passed by value.